### PR TITLE
Add initial implementation of zCVobAnimate

### PR DESCRIPTION
**Pull request by JucanAndreiDaniel**
Add initial implementation of zCVobAnimate

### CODE REVIEW

File: Assets/GothicVR/Scripts/Caches/AssetCache.cs [hunk 1]
```diff
@@ -196,7 +196,15 @@ public static IMorphMesh TryGetMmb(string key)
             if (MmbCache.TryGetValue(preparedKey, out var data))
                 return data;
 
-            var newData = new MorphMesh(GameData.Vfs, $"{preparedKey}.mmb").Cache();
+            IMorphMesh newData = null;
+            try
```
Comment (JaXt0r) [inline Assets/GothicVR/Scripts/Caches/AssetCache.cs:200]: Oh. Don't tell me there are object again, where the mmb file doesn't exist? :-)

**Review by JaXt0r — CHANGES_REQUESTED** [review]
Unfortunately I couldn't find an object which has this animation.

I also don't like we introduce certain MorphMesh parameters for PrepareMeshFilter(), yes, some of them are legacy of mine, but we need to be clear here.

I will check it during Easter and try to optimize.

File: Assets/GothicVR/Scripts/Caches/AssetCache.cs (hunk 1 above)
Comment (JucanAndreiDaniel) [reply Assets/GothicVR/Scripts/Caches/AssetCache.cs:200]: just in case 😉

**Review by JaXt0r — COMMENTED** [review]
Hey hey. I changed your wobble grass branch a little.
* The complains about AbstractMeshBuilder were more on my side. I improved the parameters a little.
* You were kind of lazy, my friend. :-D HeadMorph contains VobAnimate logic... Tststs. :rofl:  - I created a subclass for both.

Please check briefly and once ok for you, you can merge.

**Review by JaXt0r — APPROVED** [review]

**Pull request merged**